modules/nf-core/evigene/tr2aacds/main.nf

```diff
@@ -0,0 +1,104 @@
+process EVIGENE_TR2AACDS {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
Does this need process_high?

Reply:
No sure. The help section does have examples with 8 and 30 CPUs. I have now used `process_medium`. @avani-bhojwani can you please experiment with your data when possible and update the label if needed? Thanks!

Reply:
I've been using process_medium and it seems to work well